package-lock.json

```diff
@@ -1,155 +1,89 @@
 {
   "name": "rootstock-integration-tests",
   "version": "1.0.0",
-  "lockfileVersion": 3,
+  "lockfileVersion": 1,
```

Comment:
Careful

Be sure to review the code you submit

Reply:
Updated.